src/commands/init.js

```diff
@@ -0,0 +1,29 @@
+const Command = require('../base.js')
+const InitExtensionService = require('../services/initExtension')
+const initExtensionService = new InitExtensionService()
+const MarketplaceOrganizationService = require('../services/marketplaceOrganization')
+const marketplaceOrganizationService = new MarketplaceOrganizationService()
+const credentials = require('../config/credentials')
+credentials.load()
+
+class InitCommand extends Command {
+  async run () {
+    const [extension] = await Promise.all([
+      initExtensionService.promptExtensionInfo(),
+      marketplaceOrganizationService.downloadTemplate()
+    ])
+    const dynamicComponent = await initExtensionService.createDynamicComponent(
+      extension
+    )
+    initExtensionService.copyTemplateToCWD({
+      extensionType: extension.type
+    })
+    await initExtensionService.initializeManifestAccordingWithType(
+      dynamicComponent
+    )
+  }
+
+  static description = 'Inicializa um projeto Quoti'
```

Comment:
@luizAlbuquerqueJr, que tal: "Inicializa um projeto Vue para uma ou mais extensões do Quoti"?

Reply:
Ainda prefiro o meu

Reply:
Um "projeto Quoti" não significa muita coisa pra quem vai ler e o que tu tá querendo dizer com isso é exatamente criar um projeto vue com extensões do Quoti. Como não vai quebrar nada vou fazer um commit melhorando isso.
```suggestion
  static description = 'Inicializa um projeto Vue para uma ou mais extensões do Quoti'
```